frontend/src/contexts/CanvasContext.tsx

```diff
@@ -53,6 +55,35 @@ export const CanvasProvider: React.FC<{ children: React.ReactNode }> = ({ childr
       updateDimensions(obj);
     };
 
+    const handleMouseWheel = (opt: any) => {
+      const evt = opt.e;
+      if (newCanvas) {
```

Comment:
Can we please use early return?
I even have a poem for it!

One indentation less,
Makes my brain not digress
```suggestion
      if (!newCanvas) {
            return;
      };
```